changelog/bugfixes/2024-02-06-cloudsigma-remove-custom-cloudinit-service-config.md

```diff
@@ -0,0 +1 @@
+- Remove custom CloudSigma coreos-cloudinit service configuration since it will be called with the cloudsigma oem anyway. The restart of the service can also cause the serial port to be stuck in an non deterministic state which breaks future runs.
```

Comment:
```suggestion
- Removed custom CloudSigma coreos-cloudinit service configuration since it will be called with the cloudsigma oem anyway. The restart of the service can also cause the serial port to be stuck in an nondeterministic state which breaks future runs.
```